src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,255 @@
+                </v-btn>
+              </template>
+              <v-list>
+                <v-list-item class="actions-dropdown_item" @click="isEditing = true">
+                  <v-list-item-subtitle>
+                    <v-icon small>mdi-pencil</v-icon>
+                    <span class="ml-1">{{ editLabel }}</span>
+                  </v-list-item-subtitle>
+                </v-list-item>
+              </v-list>
+            </v-menu>
+          </span>
+        </v-col>
+      </v-row>
+    </v-container>
+    <v-container v-else>
+      <v-row>
+        <v-col cols="3">
+          <label class="define-company-provisions-title">Pre-existing Company Provisions</label>
+        </v-col>
+        <v-col cols="8">
+          <v-row>
+            <p class="info-text instructions-paragraph">
+              Complete this item only if the company has resolved that none of the Pre-existing Company Provisions
+              are to apply to this company (refer to Part 17 and Table 3 of the Regulation under the Business
+              Corporations Act).
+            </p>
+            <div class="d-flex align-start">
+                <v-checkbox ma-0 pa-0
+                  id="cp-checkbox"
+                  :class="{ 'invalid': isInvalid }"
+                  ref="checkboxRef"
+                  v-model="draftProvisionsRemoved"
+                />
+                <span class="info-text checkbox-label" :class="{ 'invalid': isInvalid }">
+                  The company has resolved that none of the Pre-existing Company Provisions are to apply to this
+                  company.
+                </span>
+            </div>
+          </v-row>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-container pt-10>
+          <v-row xs12>
+            <v-spacer></v-spacer>
+            <div class="action-btns">
+              <v-btn
+                large
+                color="primary"
+                id="company-provisions-done"
+                @click="setCompanyProvisionsDone()"
+              >
+                <span>Done</span>
+              </v-btn>
+              <v-btn
+                large
+                outlined
+                color="primary"
+                id="company-provisions-cancel"
+                @click="cancelCompanyProvisionChange()"
+              >
+                <span>Cancel</span>
+              </v-btn>
+            </div>
+          </v-row>
+        </v-container>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Mixins, Watch } from 'vue-property-decorator'
+import { CommonMixin } from '@/mixins'
+
+@Component({})
+export default class CompanyProvisions extends Mixins(CommonMixin) {
+  private draftProvisionsRemoved = false
+  private isEditing = false
+  private haveChanges = false
+  private originalProvisionsRemovedValue = false
+  private isInvalid = false
+  private checkboxColor = this.$vuetify.theme.currentTheme.error
```

Comment:
This is cool. I didn't know you could reference Vuetify's themes this way 👍 

But as you're not using this property, please delete it.

Reply:
👍